### PR TITLE
Fixes #906

### LINKED ISSUE #906: Text does not fit inside new assignment planning goal box
## Expected behavior (A clear and concise description of what you expected to happen) : 
working_text

The text should fit inside the box properly as seen above.

## Describe the bug (Tell us what happens instead of the expected behavior) :
not_working_text

However some text is too large and no longer fits inside the box

## Steps to Reproduce :
1. Log in as a student without a full grade progress.
2. Go to the new Assignment Planing page

### PR DESCRIPTION
**Pull request by chrisrrowland**
Fixes #906 
Adjust the css to make the field wide enough to fit the current error
text.  Not an optimal fix but works since I failed to figure out how to
get the material style to behave better.

### CODE REVIEW

**Comment by pushyamig** [comment]
@jennlove-um This change looks good, expanding the size of the text box. 
Screen Shot 2020-05-27 at 2 16 56 PM

In the issue, #906 @justin0022 also asked a question if the name should be changed or not? what is acceptable behavior?

File: assets/src/components/AssignmentGoalInput.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ function AssignmentGoalInput (props) {
           }}
           margin='normal'
           variant='outlined'
-          style={{ marginRight: '10px' }}
+          style={{ marginRight: '10px', width:'25ch' }}
```
Comment (pushyamig) [inline assets/src/components/AssignmentGoalInput.js:60]:
```suggestion
          style={{ marginRight: '10px', width: '25ch' }}
```

**Comment by pushyamig** [comment]
@chrisrrowland I have a suggestion for you in terms of spacing issue. We tend to follow the [StandardJS linting library](https://github.com/tl-its-umich-edu/my-learning-analytics/blob/master/CONTRIBUTING.md#front-end). Try to configure your IDE with that standard so that all the formatting issues with JS is taken care before committing

**Comment by pushyamig** [comment]
You might need to run the cypress test all pass. since your change modify UI and cypress test rely on previous snapshots. 
Also, make sure Jest tests also pass as well. I am not anticipating the unit tests written are that granular and nothing hurts to verify. You should find instructions in Readme for Jest test

**Comment by chrisrrowland** [comment]
ok thanks.

**Comment by jennlove-um** [comment]
I'm okay with leaving the text as is for now.  We'll get feedback from the UX testing.

**Pull request merged**